CODEOWNERS

```diff
@@ -613,6 +613,7 @@
 /subsys/portability/                      @nashif
 /lib/libc/                                @nashif
 /lib/libc/arcmwdt/                        @abrodkin @ruuddw @evgeniy-paltsev
+/misc/                                    @tejlmand
```

Comment:
A new top level directory should be considered carefully. Did you consider `/cmake/misc/` instead, to keep it hidden from users?

Reply:
Oh! How did I miss that... :)